modules/nw-external-dns-operator-logs.adoc

```diff
@@ -15,3 +15,45 @@ You can view External DNS Operator logs by using the `oc logs` command.
 ----
 $ oc logs -n external-dns-operator deployment/external-dns-operator -c external-dns-operator
 ----
+
+== External DNS Operator domain name limitations
+
+External DNS Operator uses the TXT registry which follows the new format and adds the prefix for the TXT records. This reduces the maximum length of the domain name of the DNS record for the TXT records. A DNS record cannot be present without a corresponding TXT record, so the domain name of the DNS record must follow the same limit as the TXT records.
+
```

Comment:
As just discussed, since we cannot add the links to the upstream, I think this maybe difficult for the user to understand which "new format" and "prefix" we are mentioning. How about adding the example of the record format like it's done in [the code repo's README](https://github.com/openshift/external-dns-operator/blob/d48e336597f7adb0395f28a2ec9f628e3a3d5023/README.md#length-of-the-domain-name)? 
```
DNS record: <domain-name-from-source>
TXT record: external-dns-<record-type>-<domain-name-from-source>
```
WDYT?

Reply:
good idea! 

Addressed